pkg/daemon/daemon.go

```diff
@@ -234,6 +234,8 @@ func (d *daemon) AddPeriodicUpdate() {
 					glog.Errorf("AddPeriodicUpdate(): failed to update pod annotations with err: %v", err)
 					continue
 				}
+
+				_ = d.guidPool.ReleaseGUID(guidList[index].String())
```

Comment:
if it failed just log a warning the we will know

Reply:
added